### ISSUE TITLE
This version of gosu is bringing cves

### ISSUE THREAD

**Issue by dogruis** [redis/docker-library-redis#424]
https://github.com/redis/docker-library-redis/blob/e5650da99bb377b2ed4f9f1ef993ff24729b1c16/7.4/alpine/Dockerfile#L24

https://github.com/tianon/gosu/issues/151
I created an issue to fix the cve errors

linked to https://github.com/redis/redis/issues/13663

After reading this thread I am convinced that gosu shouldn't be used at all. As the lib hasn't had a release in more than a year and the lib owner refuses to bump the golang version anytime soon to 1.23.
https://github.com/tianon/gosu/issues/136

**Comment by dogruis** [comment]
Just for the update the owner of the lib is refusing to update his library to fix CVEs as stated in his [readme](https://github.com/tianon/gosu/blob/4233b796eeb3ba76c8597a46d89eab1f116188e2/SECURITY.md).
I understand there is false positives but still maintaining libraries should be a thing.
https://github.com/tianon/gosu/issues/136

**Comment by charles-horel-rogers** [comment]
@dogruis  is there a plan to remove gosu? Or will these vulns remain present in the image

**Comment by dogruis** [comment]
I am not part of the redis team and I requested something to be done. Tbh I would not use gosu as there is command line alternatives.

**Comment by frankyjquintero** [comment]
@oranagra @sundb @enjoy-binbin tenemos alguna novedad

![Image](https://github.com/user-attachments/assets/83c1de58-614b-4c01-b15b-9aafef082172)

**Comment by tianon** [comment]
Yes, that is correct (on both counts).

**Comment by adamiBs** [comment]
@peter-sh will look into this

**Comment by adamiBs** [comment]
Our latest RC image contains the fix for this: 
https://hub.docker.com/layers/library/redis/8.0-rc1/images/sha256-4e04eab2df86d0f888262215afdf467f2509962a7c1818ac4cac9590912dfcd5

**Comment by adamiBs** [comment]
The link I sent is a docker tag that contains this fix. @dogruis

**Comment by dogruis** [comment]
Not really, it's a release candidate and not a release. So I would wait for a new release instead. Thanks a lot!!!!!

**Comment by LiorKogan** [comment]
We are very close to the 8.0 GA release.
Sorry, but we cannot share the exact date.

**Comment by rayhsieh** [comment]
Will this fix be merged into redis 6 and redis 7?
